### PR TITLE
Add updates to README

### PR DESCRIPTION
**Pull request by aemcdonald**
### What’s this PR do?  
Add updates to README & general edits

### Where should the reviewer start?  
README.md

### CODE REVIEW

**Review by npdarrington — APPROVED** [review]
Great looking gifs and verbiage!

File: README.md [hunk 1]
```diff
@@ -1,14 +1,24 @@
...
+FreshPicks is a card UI recipe application where users can view and filter recipes based on their existing pantry ingredients and recipe type. There is also a feature that allows users to favorite their recipes, "*shop*" for missing ingredients and "*cook*" a recipe, which removes any ingredients used in the cooking process.
+
+In this project, we refactored an existing code base and ensured that all functionality was working properly, created a robust testing suite, added new features such as Fetch API, SCSS, Accessibility and testing with Chai Spies.
 
-Refactor existing codebase for the app using technologies such as API Fetch, SASS, and Webpack...
-built with JS... 
 ### App in Action
 //screenshots and/or gif's here
+<img src=“assets/FitLitInAction.gif” alt=“Logo” width=“600”>
+![Filter Recipes GIF](https://recordit.co/1b386ahBlT)
```
Comment (JWoltjen) [inline README.md:17]: Saucy Gifs!!

File: README.md [hunk 2]
```diff
@@ -1,14 +1,24 @@
 # Can You Smell What's Cookin'?
 
+## Contributors
+- [Ashley McDonald](https://github.com/aemcdonald)
+- [Hanna Kim](https://github.com/hannakim91)
```
Comment (hannakim91) [inline README.md:5]: Good call on moving this to the top!

**Review by JWoltjen — APPROVED** [review]
Thanks Ashley and Hanna for taking point on the readme!!

**Pull request merged**